rosa_tutorials/rosa-mobb-security-reference.adoc

```diff
@@ -0,0 +1,507 @@
+
+[IMPORTANT]
+====
+The compliance benchmarks are continuously updated and maintained by Red Hat based on each control profile. ROSA-specific benchmarks are under development to account for the managed service components.
+====
+
+To understand and install the compliance Operator, read the Red Hat documentation:
+
+* link:https://docs.openshift.com/container-platform/4.11/security/compliance_operator/compliance-operator-installation.html[Installing the Compliance Operator]
+* link:https://docs.openshift.com/container-platform/4.11/security/compliance_operator/compliance-operator-understanding.html[Understanding the Compliance Operator]
+* link:https://docs.openshift.com/container-platform/4.11/security/compliance_operator/compliance-operator-supported-profiles.html[Supported compliance profiles]
+
+[id="rosa-mobb-security-reference-day-two-service-mesh"]
+=== OpenShift Service Mesh
+
+Red Hat OpenShift Service Mesh addresses a variety of problems in a microservice architecture by creating a centralized point of control in an application. It adds a transparent layer on existing distributed applications without requiring any changes to the application code.
+
+Red Hat OpenShift Service Mesh provides a number of key capabilities uniformly across a network of services:
+
+* **Traffic Management** - Control the flow of traffic and API calls between services, make calls more reliable, and make the network more robust in the face of adverse conditions.
+* **Service Identity and Security** - Provide services in the mesh with a verifiable identity and provide the ability to protect service traffic as it flows over networks of varying degrees of trustworthiness.
+* **Policy Enforcement** - Apply an organizational policy to the interaction between services, ensure access policies are enforced and resources are fairly distributed among consumers. Policy changes are made by configuring the mesh, not by changing application code.
+* **Telemetry** - Gain an understanding of the dependencies between services and the nature and flow of traffic between them, providing the ability to quickly identify issues.
+
+To learn more about OpenShift Service Mesh and to install the Service Mesh, read the OpenShift documentation:
+
+* link:https://docs.openshift.com/container-platform/4.11/service_mesh/v2x/ossm-architecture.html[Understanding Service Mesh]
+* link:https://docs.openshift.com/container-platform/4.11/service_mesh/v2x/installing-ossm.html[Installing the Service Mesh Operator]
+* link:https://docs.openshift.com/container-platform/4.11/service_mesh/v2x/ossm-create-mesh.html[Adding workloads to the Service Mesh]
+* link:https://docs.openshift.com/container-platform/4.11/service_mesh/v2x/ossm-security.html[Service Mesh Security]
+
+[id="rosa-mobb-security-reference-day-two-backup-and-restore"]
+=== Backup and Restore / Disaster Recovery
```

Comment:
What do you think of changing _Backup and Restore / Disaster Recovery_ to either _Backup and Restore_ **or** _Disaster Recovery_